Kernel/Bus/VirtIO/Transport/PCIe/Detect.cpp

```diff
@@ -42,6 +43,11 @@ UNMAP_AFTER_INIT void detect_pci_instances()
             // This should have been initialized by the graphics subsystem
             break;
         }
+        case PCI::DeviceID::VirtIOInput: {
+            auto& input = Input::must_create_for_pci_instance(device_identifier).leak_ref();
```

Comment:
:( Would be nice to refactor this so that some device or device manager or PCI Bus Master or something owned these objects instead of leak_ref-ing them.

Reply:
#23448 should make this a lot cleaner.